apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1310,12 +1353,10 @@ namespace MWPhysics
             else if (heightDiff < 0)
                 stats.addToFallHeight(-heightDiff);
 
+            mMutex.lock();
             mMovementResults.push_back(std::make_pair(iter->first, interpolated));
```

Comment:
It seems to me that `mMovementResults` updated only in this line (apart from cleaning before calculations) and newer used in calculations, so each thread may fill its own vector and then merge them in main thread. Then protection by a mutex not to be necessary